docs/HowTo/Interact/APIs/API.md

```diff
@@ -16,8 +16,13 @@ The following sections provide information about JSON-RPC, RPC Pub/Sub, and Grap
 
 To enable API access, use the
 [`--rpc-http-enabled`](../../../Reference/CLI/CLI-Syntax.md#rpc-http-enabled),
-[`--ws-http-enabled`](../../../Reference/CLI/CLI-Syntax.md#rpc-ws-enabled), and
-[`--graphql-http-enabled`](../../../Reference/CLI/CLI-Syntax.md#graphql-http-enabled) options.
+[`--ws-http-enabled`](../../../Reference/CLI/CLI-Syntax.md#rpc-ws-enabled),
+[`--graphql-http-enabled`](../../../Reference/CLI/CLI-Syntax.md#graphql-http-enabled), and
+`--Xrpc-ipc-enabled` options.
+
+!!! caution
+
+    `--Xrpc-ipc-enabled` is an experimental option.
```

Comment:
we don't normally doc experimental options - have we deliberately decided to do so this time?

Reply:
@macfarla, we typically don't, but I followed the precedent for other experimental features and included them here (and not the main CLI reference page) in order to document using IPC. Should I hold off until they aren't considered experimental?

Reply:
that's fine as long as we considered it